### PR TITLE
modding events and auto addition fix

### PR DESCRIPTION
**Pull request by Zychronix**

### CODE REVIEW

File: src/main/java/legend/game/modding/events/combat/SpellStatsEvent.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  public int healingPercent;
+  public int accuracy;
+  public int mpUsage;
+  public int statusChance;
+  public int element;
+  public int statusType;
+  public int buffType;
+  public int _0B;
+
+  public SpellStatsEvent(int spellId, int targetType, int _01, int specialEffect, int damageFlag, int healingPercent, int accuracy, int mpUsage, int statusChance, int element, int statusType, int buffType, int _0B) {
```
Comment (LordMonoxide) [inline src/main/java/legend/game/modding/events/combat/SpellStatsEvent.java:20]: Finalize vars and lowercase both `_0bs`

File: src/main/java/legend/game/modding/events/inventory/ShopItemEvent.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package legend.game.modding.events.inventory;
+
+import legend.game.modding.events.Event;
+
+public class ShopItemEvent extends Event {
+  public final int shopId;
+  public final int slotId;
+  public int itemId;
+
+  public ShopItemEvent(int shopId, int slotId, int itemId) {
```
Comment (LordMonoxide) [inline src/main/java/legend/game/modding/events/inventory/ShopItemEvent.java:10]: Finalize vars

**Review by LordMonoxide — CHANGES_REQUESTED** [review]

File: src/main/java/legend/game/modding/events/combat/SpellStatsEvent.java (hunk 1 above)
Comment (Zychronix) [reply src/main/java/legend/game/modding/events/combat/SpellStatsEvent.java]: vars finalized and 0b is now in lowercase

File: src/main/java/legend/game/modding/events/inventory/ShopItemEvent.java (hunk 2 above)
Comment (Zychronix) [reply src/main/java/legend/game/modding/events/inventory/ShopItemEvent.java]: vars finalized

**Review by LordMonoxide — APPROVED** [review]

**Pull request merged**